.changeset/tricky-paws-march.md

```diff
@@ -0,0 +1,5 @@
+---
+"@comet/admin": major
+---
+
+Remove `endAdornment` prop from `FinalFormSelect` component.
```

Comment:
@jamesricky should we add more explanation why we're doing this? What are the concrete reasons? MUI doesn't support it in their select, there's already clearable and the chevron?

Reply:
Yes, that would be helpful 👍🏼 

I think the reason was, that it didn't work properly (the end-adornment overlayed the select arrow or something) and since MUI does not support this on their Select we decided to also remove support for that. 

Is that correct @thomasdax98?

Reply:
The reason comes from this PR https://github.com/vivid-planet/comet/pull/1363:

> Since the end-adornment for the clear-button needs absolute positioning, it is no longer possible to use the clearable prop in combination with an endAdornment. However, this is not a common use-case, and an endAdornment on a select is generally not supported by MUI, see: https://github.com/mui/material-ui/issues/17799.

Followed by the discussion https://github.com/vivid-planet/comet/pull/1363#issuecomment-1809786877:

>> > > However, this is not a common use-case, and an endAdornment on a select is generally not supported by MUI, see: [mui/material-ui#17799](https://github.com/mui/material-ui/issues/17799).
>> > 
> >> 
> >> Why don't we remove support for `endAdornment` for selects as well? I can't think of a likely use case anyway
>> 
>> I can't really think of a good use-case either... However, there might be some older projects, that manually implement the Clear-Button as an `endAdornment`, from before the `clearable` Prop existed. That should be considered a breaking change imo and wait for `v5` or `v6`.
>
>Yes, of course, removing it completely would be a breaking change. @thomasdax98 could you please add a task for this future improvement? Thanks!

So yes, the reason was that endAdornment and clearable need absolute positioning to not overlap the arrow. So we can't have both at the same time -> we decided to remove endAdornment because MUI doesn't support it and there are no good use cases